ui/js/models/perf/alerts.js

```diff
@@ -34,6 +34,33 @@ treeherder.factory('PhAlerts', [
 
             return url;
         };
+        Alert.prototype.getSubtestsURL = function(alertSummary) {
+            function formURLParameters(associativeArray) {
+                var parameters = [];
+                for (var key in associativeArray) {
+                    if (associativeArray.hasOwnProperty(key)) {
+                        parameters.push(key + '=' + encodeURIComponent(associativeArray[key]));
+                    }
+                }
+                return parameters.join('&');
+            }
```

Comment:
It's probably not necessary to define a function for this.

Something like this should work, if params is just an object:

```js
_.map(params, function(v, k) { return k + "=" + v; }).join('&');
```